README.md

```diff
@@ -294,6 +295,30 @@ const externalSchema = {
 const stringify = fastJson(schema, { schema: externalSchema })
 ```
 
+<a name="long"></a>
+#### Long integers
+Long integers (64-bit) are supported using the [long](https://github.com/dcodeIO/long.js) module.
+Example:
+```javascript
+const Long = require('long')
+
+const stringify = fastJson({
+  title: 'Example Schema',
+  type: 'object',
+  properties: {
+    id: {
+      type: 'integer'
```

Comment:
I cannot find actual documentation on the data types on the official website, but there are examples in the specification that are using the `integer` type. It is also already used in the current [README.md](https://github.com/fastify/fast-json-stringify/blob/master/README.md#api)